spec/system/provider_interface/setup_provider_relationship_permissions_spec.rb

```diff
@@ -0,0 +1,129 @@
+require 'rails_helper'
+
+RSpec.feature 'Setting up provider relationship permissions' do
+  include DfESignInHelpers
+
+  scenario 'Provider user sets up permissions for their organisation' do
+    given_i_am_a_provider_user_with_dfe_sign_in
+    and_the_provider_permissions_feature_is_enabled
+    and_i_can_manage_organisations
+    and_my_organisations_have_not_had_permissions_setup
+
+    when_i_sign_in_to_the_provider_interface
+    then_i_can_see_the_permissions_setup_page
+
+    when_i_click_continue
+    and_i_choose_permissions_for_the_provider_relationship
+    and_i_confirm_my_choices
+    then_i_see_permissions_were_successfully_saved
+
+    when_i_click_continue
+    then_i_can_see_the_permissions_setup_page
+
+    when_i_click_continue
+    and_i_choose_permissions_for_another_provider_relationship
```

Comment:
Maybe set up the test data so that the user only has 'Manage organisations' for one of the providers? This way we'll test the specificity of 'Manage organisations' instead of the ability to handle multiple sets of org permissions. As I understand this, this is an asymmetric feature, in that the training provider is the one deciding to give access to the ratifying provider and not the other way around.

You could always add a third provider, of course, and test both the specificity of the permission and the ability to handle multiple sets of permissions (for multiple providers).

Reply:
https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/2231/commits/64852d66c3378a87206797339490eb6617697632#diff-29880b76c764d0e6550048eef5418477R49 :+1: I've removed the ratifying provider from the providers associated with the user.